src/sphinx_basic_ng/theme/basic-ng/theme.conf

```diff
@@ -4,3 +4,12 @@ stylesheet = basic.css
 sidebars =
 
 [options]
+announcement = 
+topbar_left = 
+topbar_middle = 
+topbar_right = 
+middle_content = "components/page-content.html"
+sidebar_left = "components/site-logo.html, components/site-nav.html"
```

Comment:
There shouldn't be a need for quotes around this.
In fact, there probably shouldn't be any quotes at all in `conf.py`.

I'm not sure about the path separators, though, I've never used sub-directories ...